R/drive_publish.R

```diff
@@ -40,21 +40,38 @@ drive_change_publish <- function(file,
                                  publish = TRUE,
                                  ...,
                                  verbose = TRUE) {
-  file_update <- drive_is_published(file = file, verbose = FALSE)
-
-  file_update <- confirm_single_file(file_update)
+  file_update <- drive_show_publish(file = file, verbose = FALSE)
+  file_update <- confirm_some_files(file_update)
+  file_update <- split(file_update, 1:nrow(file_update))
```

Comment:
Ah probably `pmap()`, or `map()` on `drive_resource` would be better. ⚠️ you are going to see `split()` a few times since I tried to stay somewhat consistent, but I will systematically change those on my next push.